src/qt/optionsmodel.cpp

```diff
@@ -789,7 +776,6 @@ void OptionsModel::checkAndMigrate()
     migrate_setting(SpendZeroConfChange, "bSpendZeroConfChange");
     migrate_setting(ExternalSignerPath, "external_signer_path");
 #endif
-    migrate_setting(MapPortUPnP, "fUseUPnP");
```

Comment:
In commit "qt: remove UPnP settings" (844770b05ebc34789dc46d70cd6398089539c915)

It probably would be better to replace this line with:

```c++
if (settings.contains("fUseUPnP")) {
    if (node().getPersistentSetting("upnp").isNull()) UpdateRwSetting(node(), "upnp", "", settings.value("fUseUPnP").toBool());
    settings.remove("fUseUPnP");
}
```

So the setting would still be migrated and a warning could be logged if it was enabled.